atef/widgets/config.py

```diff
@@ -1865,16 +1865,30 @@ def eventFilter(self, object: QLineEdit, event: QEvent) -> bool:
         if not isinstance(object, QLineEdit):
             return False
         if event.type() == QEvent.FocusIn:
-            object.setFrame(True)
-            object.setReadOnly(False)
+            self.set_edit_style(object)
             return True
         if event.type() == QEvent.FocusOut:
-            if object.text():
-                object.setFrame(False)
-            object.setReadOnly(True)
+            self.set_no_edit_style(object)
             return True
         return False
 
+    @staticmethod
+    def set_edit_style(object: QLineEdit):
```

Comment:
Docstrings would be nice if you have a chance

Reply:
I'll add some docstrings